HW2/app/src/main/java/com/example/milab_hw2/StarksGalleryActivity.java

```diff
@@ -0,0 +1,20 @@
+package com.example.milab_hw2;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+
+public class StarksGalleryActivity extends GalleryActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_starks_gallery);
+
+        String[] starks_data= {"Arya", "And such"};
+        int[] images = {}; //To be implemented
```

Comment:
Why is it to be implemented? it is part of the task

Reply:
Seems like I mistakenly pushed an earlier debug version. I am pushing the finished version instead